mantidproject/mslice
# Add error messages to calculate projection

Previously, misuse of the calculate projections button would give errors in the console but not report anything to the user through the GUI. Now, messages are displayed if no workspace is selected or a workspace is the wrong type.

**To test:**
- Click calculate projection without selecting a workspace
- Observe error message

And:

- Select a workspace that is not a workspace2D (e.g. _QE)
- Click calculate projection
- Observe error message

Fixes #38
